src/time/forward.h

```diff
@@ -13,6 +13,7 @@
 
```

Comment:
Please update the header guard.

Reply:
Done in [d53ffe3](https://github.com/squid-cache/squid/pull/1001/commits/d53ffe3f66a8db05d3cc6327960a50594d1a80be)